cmd/gossamer/main.go

```diff
@@ -207,8 +207,8 @@ func importRuntimeAction(ctx *cli.Context) error {
 // then creates and starts the node and node services
 func gossamerAction(ctx *cli.Context) error {
 	// check for unknown command arguments
-	if arguments := ctx.Args(); len(arguments) > 0 {
-		return fmt.Errorf("failed to read command argument: %q", arguments[0])
+	if arguments := ctx.Args(); arguments.Len() > 0 {
+		return fmt.Errorf("failed to read command argument: %q", arguments.Get(0))
```

Comment:
nit whilst we're at it changing the line:

```suggestion
		return fmt.Errorf("%d extra unknown command line arguments", arguments.Len())
```

Ideally also:

- List all unknown arguments (not just the number of them, nor the first one only)
- Have a sentinel error; but since we're in main.go I guess it's okay to have just a formatted string.